### ISSUE TITLE
.vscode should be in .gitignoore

### ISSUE THREAD

**Issue by vybhav72954** [jhochwald/PowerShell-collection#7]
`.vscode` an IDE gen file, is not needed in the Project Repo, it is safe to put it in `.gitignore`

**Comment by jhochwald** [comment]
Hi @vybhav72954,

interesting, the following is part of my ignore File:
```
# Visual Studio Code
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```
Looks like this is ignored!

**Comment by jhochwald** [comment]
Fixed with 00348e4
The gitignore.io reference seems to have an issue :(
Instead of just `.vscode/*` I also had to add `.vscode`

Thanks @vybhav72954 for the hint!
